util/prepare_environment/060_build_corelibs_libdispatch.sh

```diff
@@ -27,6 +29,8 @@ pushd $SWIFT_ANDROID_SOURCE
 				--disable-build-tests \
 				--prefix=$SWIFT_INSTALLATION_PATH/usr
 
+		sed -ie "s@-L/tmp/placeholder@-L$SWIFT_INSTALLATION_PATH/usr/lib/swift/android -lswiftCore -L$NDK/toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/arm-linux-androideabi/lib/armv7-a -latomic@" src/Makefile
```

Comment:
I think you can pass it directly to configure without seding. Via LD_FLAGS
Like here https://github.com/zayass/swift-android-vagrant/blob/master/corelibs/provision/010_build_corelibs_libdispatch.sh#L16